### PR TITLE
l10n: Correct text string

### PR DESCRIPTION
**Pull request by Valdnet**
Signed-off-by: Valdnet <47037905+Valdnet@users.noreply.github.com>

### CODE REVIEW

**Comment by dartcafe** [comment]
Shouldn't it be an exclamation point as it is an imperativ?

**Comment by Valdnet** [comment]
Not. This is just information.

**Comment by rakekniven** [comment]
Having a dot of the end or not is a huge task by seing the complete Nextcloud project.
There are hundreds, or more messages without following a rule and being so or so.

In my option we should define the rules and modifing strings afterwards by reworking/reviewing complete resources (apps). One after the other.

File: src/js/components/Settings/AdminShareSettings.vue [hunk 1]
```diff
@@ -38,12 +38,12 @@
 				:preserve-search="true"
 				:multiple="true"
 				:loading="isLoading"
-				:placeholder="t('polls', 'Leave empty to disallow for all')"
+				:placeholder="t('polls', 'Leave empty to disallow for all.')"
 				@search-change="loadGroups" />
 		</div>
 
 		<CheckboxRadioSwitch :checked.sync="allAccessLimited" type="switch">
-			{{ t('polls', 'Disallow publishing polls to all users.') }}
```
Comment (Valdnet) [inline src/js/components/Settings/AdminShareSettings.vue:46]: This should definitely be validated to match the rest of the text strings in the settings for this app.

File: src/js/components/Settings/AdminShareSettings.vue [hunk 2]
```diff
@@ -38,12 +38,12 @@
...
+			{{ t('polls', 'Disallow publishing poll to all users') }}
```
Comment (Valdnet) [inline src/js/components/Settings/AdminShareSettings.vue:46]: This should definitely be validated to match the rest of the text strings in the settings for this app.

**Comment by dartcafe** [comment]
Proposal: @Valdnet I'll add you as a reviewer before merging PR with new/changed static texts.

**Comment by Valdnet** [comment]
> I'll add you as a reviewer before merging PR with new/changed static texts.

@dartcafe Okay 😉.

**Review by dartcafe — APPROVED** [review]

**Pull request merged**